docs/examples/irf_tool_config.json

```diff
@@ -25,11 +25,11 @@
     "allowed_tels": [1]
   },
   "DataBinning": {
-    "true_energy_min": 0.01,
-    "true_energy_max": 100,
+    "true_energy_min": 0.005,
+    "true_energy_max": 200,
     "true_energy_n_bins_per_decade": 5,
-    "reco_energy_min": 0.01,
-    "reco_energy_max": 100,
+    "reco_energy_min": 0.005,
+    "reco_energy_max": 200,
```

Comment:
Is 200 TeV a reasonable upper limit or will this just result in empty bins to high energies?

Reply:
The new MC with higher zenith pointings have a maximum energy of 200 TeV, so this range accommodates them.
We will have empty bins for MC with different sky pointing nodes in either low or high energies, when compared together, for example, in performing IRF interpolations. 

But so far, I have not seen a problem with having empty bins in the IRFs. If you can point to some issues, it would be great to look into them.